kiwi/system/setup.py

```diff
@@ -95,6 +96,9 @@ def import_description(self):
         self._import_custom_archives()
         self._import_cdroot_archive()
 
+    def script_exists(self, name):
+        return os.path.exists(self.description_dir + '/' + name)
```

Comment:
@schaefi why is that string concats all over the place? A typical way would be:

```python
def script_exists(self, name:str) -> bool:
    """
    I am documentation and this would be a failure, if PyLint would be in use.
    """
    return os.path.exists(os.path.join(self.description_dir, name))
```

Reply:
I had some not so nice experience with os.path.join in the past. But let's assume this is no longer an issue with py3

Reply:
@schaefi what problems, BTW? I actually never had any issues, but you need to be careful with slashes:

```
>>> os.path.join("this", "is", "/wrong")
'/wrong'
```
But this is also works as designed:

```
join(a, *p)
    Join two or more pathname components, inserting '/' as needed.
    If any component is an absolute path, all previous path components
    will be discarded.  An empty last part will result in a path that
    ends with a separator.
```

Reply:
I think I had some trouble with path.join on relative references. Something like "/foo/bar/../baz". But this is long time ago and I really can't remember good enough to open a conversation about it. I'm sure it behaved as designed but caused me some headaches.

Reply:
> as long as you get rid of those strings concats, this should do

yep we care for this, however I think you will find code that still does this. Whenever it comes up we fix it :)

Reply:
not here please :) that PR already has grown too big for more cleanup/refactoring